src/components/Settings/ShareOnSocialMedia.js

```diff
@@ -55,56 +56,91 @@ const TelegramShareButton = styled(_TelegramShareButton)`
   ${commonIconStyle};
 `;
 
-const ShareOnSocialMedia = () => {
-  const shareUrl = 'https://susi.ai';
-  const title = 'Lets chat with SUSI.AI, the open source personal assistant';
-  const shareText =
-    'Lets chat with SUSI.AI, the open source personal assistant https://susi.ai';
+const shareUrl = 'https://susi.ai';
+const shareText =
+  'Lets chat with SUSI.AI, the open source personal assistant https://susi.ai';
 
-  return (
-    <React.Fragment>
-      <DialogTitle>Share about SUSI</DialogTitle>
+class ShareOnSocialMedia extends Component {
+  state = {
+    shareUrl,
+    title:
+      this.props.title ||
+      'Lets chat with SUSI.AI, the open source personal assistant',
+    shareText,
+  };
 
-      <CopyWithButton value={shareText} />
-      <ShareIconContainer>
-        <IconButton>
-          <FacebookShareButton url={shareUrl} quote={title}>
-            <div>
-              <FacebookIcon size={42} />
-            </div>
-          </FacebookShareButton>
-        </IconButton>
-        <IconButton>
-          <TwitterShareButton url={shareUrl} title={title}>
-            <div>
-              <TwitterIcon size={42} />
-            </div>
-          </TwitterShareButton>
-        </IconButton>
-        <IconButton>
-          <LinkedinShareButton url={shareUrl} title={title}>
-            <div>
-              <LinkedinIcon size={42} />
-            </div>
-          </LinkedinShareButton>
-        </IconButton>
-        <IconButton>
-          <WhatsappShareButton url={shareUrl} title={title}>
-            <div>
-              <WhatsappIcon size={42} />
-            </div>
-          </WhatsappShareButton>
-        </IconButton>
-        <IconButton>
-          <TelegramShareButton url={shareUrl} title={title}>
-            <div>
-              <TelegramIcon size={42} />
-            </div>
-          </TelegramShareButton>
-        </IconButton>
-      </ShareIconContainer>
-    </React.Fragment>
-  );
+  handleChange = event => this.setState({ shareText: event.target.value });
+
+  render() {
+    return (
+      <React.Fragment>
+        <DialogTitle>Share about SUSi.AI</DialogTitle>
```

Comment:
It should be `SUSI.AI`. That was a typo in the previous comment. 

Also, you must think logically before following any feedback.